storybook/src/components/logo/logo.core-template.ts

```diff
@@ -0,0 +1,13 @@
+import { Logo } from "dso-toolkit";
+import { html } from "lit-html";
+import { ComponentImplementation } from "../../templates";
+import { ifDefined } from "lit-html/directives/if-defined.js";
+
+export const coreLogo: ComponentImplementation<Logo> = {
+  component: "logo",
+  implementation: "core",
+  template: () =>
+    function logoTemplate({ label, ribbon }) {
+      return html`<dso-logo label=${ifDefined(label)} ribbon=${ifDefined(ribbon)}></dso-logo> `;
```

Comment:
Daar staat een spatie aan het einde die we daar niet per se nodig hebben, maar waar we op andere plekken ons zomaar kunnen afvragen "hoe komt die spatie daar nou?".